hack/build-rpm-release.sh

```diff
@@ -53,5 +53,21 @@ make clean
 # migrate the tito artifacts to the Origin directory
 mkdir -p "${OS_OUTPUT}"
 cp -r "${tito_output_directory}"/* "${OS_OUTPUT}"
-mkdir -p "${OS_LOCAL_RELEASEPATH}"
-cp "${tito_tmp_dir}"/x86_64/*.rpm "${OS_LOCAL_RELEASEPATH}"
\ No newline at end of file
+mkdir -p "${OS_LOCAL_RELEASEPATH}/rpms"
+cp "${tito_tmp_dir}"/x86_64/*.rpm "${OS_LOCAL_RELEASEPATH}/rpms"
```

Comment:
RPMs are placed flat in the output from `tito`, and `createrepo` below is what is actually building the repo and metadata